src/components/Dropdown/Items/MultiSelectItem/MultiSelectItem.tsx

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable @typescript-eslint/no-unnecessary-type-constraint */
 import React from 'react';
```

Comment:
Priority is not aligned well:
![Screenshot 2024-11-08 at 11 25 55](https://github.com/user-attachments/assets/2062b6d4-5d24-4a55-aedb-3834c7d50c98)

Reply:
The clear Icon also doesn't seem to be perfectly align, but it is really minor thing. What is noticeable is that all the icons are too small, especially priority icon which is 12x12 ox and in design 20x20. The arrow one has onle 2 px difference
<img width="287" alt="Screenshot 2024-11-08 at 13 34 11" src="https://github.com/user-attachments/assets/70f9cd28-e82c-4825-9146-0b36c3f81e7c">
<img width="93" alt="Screenshot 2024-11-08 at 13 31 44" src="https://github.com/user
<img width="59" alt="Screenshot 2024-11-08 at 13 31 37" src="https://github.com/user-attachments/assets/f6a94d65-6bd2-4a28-93e2-3b33875198d1">
-attachments/assets/40b6f9ba-9664-4c2f-ad75-18c87944a573">
<img width="66" alt="Screenshot 2024-11-08 at 13 31 32" src="https://github.com/user-attachments/assets/60c5a093-2d09-406e-a410-629524d937f4">

Reply:
the Clear one also should be 16x16
 ` <Clear aria-label={clearLabel} viewBox="0 0 24 24" height="14px" width="14px" />`

Reply:
The Prioirty button icon is coming from IconButton. I used the small, since the normal size is bigger then the pill button all together. Not sure where to go with it. Dima needs to align his designs :-D 
I'll change the rest

Reply:
<img width="706" alt="Screenshot 2024-11-08 at 15 26 01" src="https://github.com/user-attachments/assets/9ad75e2e-2fa2-49e5-8626-8a92a72cad74">

after making the icons 16px, I think the alignment is now ok, no?

Reply:
I think it should be some space between to button and text:
![Screenshot 2024-11-08 at 15 29 39](https://github.com/user-attachments/assets/4599cf40-4b53-4d1a-9664-9fd9092cbf78)

Reply:
yeah, with 16px, looks great. And agree with @mukiienko , some space would be nice 😊

Reply:
I will ask Dima to make a design proper with the priority icon, using the actual button icon. I didn't add the space, because... whatever, I'm not the designer, and I don't like to make up things :-D